operator_ui/src/api/v2/ocrKeys.ts

```diff
@@ -0,0 +1,62 @@
+import * as jsonapi from '@chainlink/json-api-client'
+import { boundMethod } from 'autobind-decorator'
+import * as models from 'core/store/models'
+import * as presenters from 'core/store/presenters'
+/**
+ * Create adds validates, saves a new OcrKey.
+ *
+ * @example "<application>/off_chain_reporting_keys"
+ */
+const CREATE_ENDPOINT = '/v2/off_chain_reporting_keys'
```

Comment:
Could we
```suggestion
const CREATE_ENDPOINT = INDEX_ENDPOINT 
```
? Or not have it all as it's redundant in RESTful APIs.